pyproject.toml

```diff
@@ -99,6 +99,7 @@ module = [
   "google.resumable_media.common.*",
   "google.rpc.status_pb2.*",
   "google_crc32c.*",
+  "grpc.*",
```

Comment:
grpc has a stubs package I believe -- that should fix this without regressing it?

that said grpc doesn't really play well with python -- and we don't currently use grpc at sentry for service-to-service communication.  was there a design review to decide on this as the service-to-service technology?